code/game/objects/items/weapons/shields.dm

```diff
@@ -624,3 +624,50 @@
 		set_light(1.5, 1.5, COLOR_LIGHTING_RED_BRIGHT)
 	else
 		set_light(0)
+		
+/obj/item/shield/parrying
+	name = "parrying dagger"
+	desc = "A parrying dagger forged in a specific way to easily parry other melee attacks with ease, sacrificing its lethality in exchange."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "dagger"
+	item_state = "dagger"
+	flags = CONDUCT
+	w_class = ITEM_SIZE_SMALL
+	slot_flags = SLOT_BELT
+	force = WEAPON_FORCE_NORMAL
+	throwforce = WEAPON_FORCE_DANGEROUS
+	throw_speed = 2
+	throw_range = 6
+	armor_list = list(melee = 0, bullet = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	matter = list(MATERIAL_PLASTEEL = 5, MATERIAL_PLASTIC = 2)
+	price_tag = 250
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	max_durability = 100 //So we can brake and need healing time to time
+	durability = 100
```

Comment:
```suggestion
	max_durability = 35 //So we can brake and need healing time to time
	durability = 35
```
Less insainly good values as these are cheap to make and have tons of uses and are small